src/main/java/org/stroganov/dao/JsonLibraryDAO.java

```diff
@@ -48,7 +48,7 @@ public boolean addBook(Book book) throws IOException {
     }
 
     @Override
-    public boolean addBook(List<Book> bookList) throws IOException {
+    public boolean addBookList(List<Book> bookList) throws IOException {
```

Comment:
You do not need to specify the name of the data structure in the name of the method.
You can replace the list with a larger abstraction level ( Optional)
```suggestion
    public boolean addBooks(Collection<Book> bookList) throws IOException {
```

Reply:
fixed